src/components/field/index.js

```diff
@@ -88,8 +88,8 @@ export default class TextField extends PureComponent {
 
     renderAccessory: PropTypes.func,
 
-    prefix: PropTypes.string,
-    suffix: PropTypes.string,
```

Comment:
`PropTypes.string` will throw an error as `renderAffix` function returns the `Affix` component.
In which `prefix|suffix` is inside a `<Animated.Text style={[style, textStyle]}>{children}</Animated.Text>` (it's the children var)

Reply:
In lines 319-320 (the `renderAffix` func) you should check if `prefix|suffix` is a valid element and return it instead of the `<Affix />` component:

E.g.
```jsx
{React.isValidElement(affix)
  ? affix // Return the custom element
  : (<Affix style={affixTextStyle} {...props}>{affix}</Affix>)
}
```